### DISCUSSION TITLE
Is there any possibility to save the WebRTC stream to FTP server.

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#2137] [Questions] [answered]
Thanks for the fantastic product.

Just I wanted to know that is their any possible to save the WebRTC stream to the FTP server. 
If no, can you please let me know any other alternatives for the this type of issues.

Once again I would like to thank you for the fantastic product.

One more issue is I placed a Jetson Nano Developer Kit at my office and there are 30 cameras. With the help of Jetson and MediaMTX I am streaming and monitoring the stream. I observed that Stream is stopping after 4-5 Hours and when we restart the Docker it is streaming again, For the this we brought a instance in the AWS and changed google Turn Server into our own AWS Turn Server but, no use . Could you please help me.

Thank you.

**Comment by aler9** [comment, marked as answer]
Hello,

> Just I wanted to know that is their any possible to save the WebRTC stream to the FTP server.

You can save streams on disk by following instructions in the README:

https://github.com/bluenviron/mediamtx#save-streams-on-disk

After saving the streams to disk, you can synchronize the folder with your remote FTP server.

> One more issue is I placed a Jetson Nano Developer Kit at my office and there are 30 cameras. With the help of Jetson and MediaMTX I am streaming and monitoring the stream. I observed that Stream is stopping after 4-5 Hours 

Create an issue and attach server logs.

[a comment by an automated account is not shown]
